changelogs/current.yaml

```
@@ -5,6 +5,10 @@ behavior_changes:
 - area: build
   change: |
     official released binary is now built on Ubuntu 20.04, requires glibc >= 2.30.
+- area: listener
+  change: |
+    Previously the listener update with different :ref:`socket_options <envoy_v3_api_field_config.listener.v3.Listener.socket_options>` will be ignored. Now, when the :ref:`socket_options <envoy_v3_api_field_config.listener.v3.Listener.socket_options>` updated, a new socket will created for the listener and
+    apply the new :ref:`socket_options <envoy_v3_api_field_config.listener.v3.Listener.socket_options>` to it.
```

```suggestion
    Previously a listener update with different :ref:`socket_options <envoy_v3_api_field_config.listener.v3.Listener.socket_options>` was ignored. Now, when the :ref:`socket_options <envoy_v3_api_field_config.listener.v3.Listener.socket_options>` are updated, a new socket will be created for the listener and the new :ref:`socket_options <envoy_v3_api_field_config.listener.v3.Listener.socket_options>` applied to it. This only happens when reuse_port is true. Otherwise if socket options change the update is NACKd.
```

Similar note about the runtime flag.